### PR TITLE
Update subsurface from 5.0.5 to 5.0.6

### PR DESCRIPTION
**Pull request by Porkepix**
Created with `brew bump-cask-pr`.

### CODE REVIEW

File: Casks/subsurface.rb [hunk 1]
```diff
@@ -1,14 +1,14 @@
 cask "subsurface" do
-  version "5.0.5"
+  version "5.0.6"
 
   if MacOS.version <= :mojave
     sha256 "d33cc7bd277be1559e67e5bd08d512c0a901430dd2685df278e06fa17c21f3c3"
     url "https://subsurface-divelog.org/downloads/Subsurface-#{version}-10.13+14.dmg"
```
Comment (miccal) [inline Casks/subsurface.rb:6]: This one needs updating also.

Comment (Porkepix) [reply Casks/subsurface.rb:6]: Done. However I find it surprising the `brew bump-cask-pr` command update not one but two hashes, but fails to do the third?

Comment (miccal) [reply Casks/subsurface.rb]: `brew bump-cask-pr` cannot handle all possible cases of MacOS versions and CPU architectures, which is why it is important to check your PR's and update manually if required.

Comment (Porkepix) [reply Casks/subsurface.rb]: @miccal Why would the command miss one of the hashes to update?

**Pull request merged**